charts/opentelemetry-demo/values.yaml

```diff
@@ -598,7 +597,8 @@ components:
       nodeSelector: {}
       affinity: {}
       tolerations: []
-
+    command: []
```

Comment:
When #557 lands, this would make sense to be included in the sample documented component at the top of the components section instead of inline within just the redis component.

Reply:
#557 is merged now. After a rebase, please move these 2 parameters into the commented example at the top of the components section, adding comments about how to use them.